EventListener/AttachRefreshTokenOnSuccessListener.php

```diff
@@ -77,6 +84,7 @@ public function __construct(
         $this->singleUse = $singleUse;
         $this->refreshTokenGenerator = $refreshTokenGenerator;
         $this->extractor = $extractor;
+        $this->cookie = $cookie;
```

Comment:
Maybe this should have a default structure set (similar to how the 5.3 authenticator sets defaults for its config array in the constructor)?